### PR TITLE
Media: Add `decoding=async` to all images

### LINKED ISSUE #9740: Media: Add `decoding=async` to all images
## Feature Description

Background [Trac #53232](https://core.trac.wordpress.org/ticket/53232). 

In WordPress core, there is a plans to add `decoding=async` to all images. This improves page render performance. We should consider doing the same for images found in the media library.

### PR DESCRIPTION
**Pull request by spacedmonkey**
## Summary

Add `decoding=async` to all images rendered in the web stories editor. By adding this attribute, is improves rendering and time to first paint. See user facing change to compare results. 

## User-facing changes

|Request|Before  | After |
--- | --- | ---|
|Normal|main|branch|
|Slow CPU|main-slow-cpu|branch-slow-cpu|
|Fast 3G|main-fast-3g|branch-fast-3g|

## Testing Instructions

- [x] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This code is covered by automated tests (unit, integration, and/or e2e) to verify it works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #9740

### CODE REVIEW

File: packages/story-editor/src/elements/media/imageDisplay.js [hunk 1]
```diff
@@ -76,6 +76,7 @@ function ImageDisplay({ element, box, previewMode }) {
   );
 
   imgProps.crossOrigin = 'anonymous';
+  imgProps.decoding = 'async';
```
Comment (swissspidy) [inline packages/story-editor/src/elements/media/imageDisplay.js]: FWIW, this overrides `decoding="sync"`

Comment (spacedmonkey) [reply packages/story-editor/src/elements/media/imageDisplay.js]:
```suggestion
```

**Comment by spacedmonkey** [comment]
@swissspidy What are your recommendations on profiling this? I have tried react profiler and lighthouse, but because the image rendering in after the editor is loaded, I can't get a good reading on them. Async used in places, where height and width are set and no need to calulated. In all cases I have highlighted in our codebase, these values are set. 

How much faster does it have to be to get this merged? Is it okay, we just believe it will be faster?

**Comment by swissspidy** [comment]
https://github.com/google/web-stories-wp/issues/9740#issuecomment-981927414 mentioned to enable CPU throttling before measuring LCP.

Regardless of any numbers, are there any changes _visible_ to the user? Can you _see_ any changes with CPU throttling enabled?

**Comment by spacedmonkey** [comment]
I have done some testing. See the before and after results. The best result is in time to first paint. Meaning this is a meaningful and worthwhile change.

**Comment by swissspidy** [comment]
Is the trend still there even if you run the tests a number of times?

**Comment by spacedmonkey** [comment]
> Is the trend still there even if you run the tests a number of times?

Each one of these tests were run 3 times

**Review by swissspidy — APPROVED** [review]

**Pull request merged**